packit_service/service/api/webhooks.py

```diff
@@ -96,7 +96,11 @@ def post(self):
 
         celery_app.send_task(
             name=getenv("CELERY_MAIN_TASK_NAME") or CELERY_DEFAULT_MAIN_TASK_NAME,
-            kwargs={"event": msg},
+            kwargs={
+                "event": msg,
+                "source": "github",
+                "event_type": request.headers.get("X-GitHub-Event"),
```

Comment:
sweet! you're right it's amazing we know this up front and can decide without spawning the celery tasks

Reply:
#2066 is about spawning fewer celery tasks. This one is about not calling all the [Parser.parse_xyz_event()](https://github.com/packit/packit-service/blob/main/packit_service/worker/parser.py#L132) to realize what type of event we're parsing when we can get the info from outside and then call only one of the `Parser.parse_xyz_event()` to parse it. But definitely a good improvement.